packages/core/lib/commands/commands.js

```diff
@@ -26,3 +25,31 @@ module.exports = [
   "version",
   "watch"
 ];
+
+//Subset of truffle commands that are allowed to run in console REPLS.
+//Excluded commands are:
+//  console, dashboard, db, develop, init and watch
+const validTruffleConsoleCommands = [
```

Comment:
Not sure it matters much but IMO it would be better to have an excluded commands list (which this could then perhaps be built from) because consider what happens when someone needs to add something.  Excluded commands are the exceptional case, right?  So they're the ones that should get "you need to update two things for this".

Reply:
Good point. I wanted to remove duplicated derivation logic with this PR. I could see maintaining an exclusion list in this file, using it to derive `validTruffleConsoleCommands` once. Would that be ok?

Reply:
Sounds good to me.